src/xia2/Wrappers/Dials/Cosym.py

```diff
@@ -67,6 +68,9 @@ def set_best_monoclinic_beta(self, best_monoclinic_beta):
         def set_lattice_symmetry_max_delta(self, lattice_symmetry_max_delta):
             self._lattice_symmetry_max_delta = lattice_symmetry_max_delta
 
+        def set_reference_pdb(self, reference_pdb):
```

Comment:
I think it would be clearer to call this property and associated methods `reference_file` rather than `reference_pdb`, as we support pdb, mtz, cif...

Reply:
Done :) Will be located on the separated branch.